UnitTests/GitUITests/ThreadHelperTests.cs

```diff
@@ -241,7 +241,7 @@ private static void JoinPendingOperations()
             // Note that ThreadHelper.JoinableTaskContext.Factory must be used to bypass the default behavior of
             // ThreadHelper.JoinableTaskFactory since the latter adds new tasks to the collection and would therefore
             // never complete.
-            ThreadHelper.JoinableTaskContext?.Factory.Run(() => ThreadHelper.JoinPendingOperationsAsync());
+            ThreadHelper.JoinableTaskContext.Factory.Run(() => ThreadHelper.JoinPendingOperationsAsync());
```

Comment:
❗️ This null check also cannot change

Reply:
I strongly disagree for the same reason as for `AsyncTestHelper.WaitForPendingOperations` above and all `ThreadHelperTests` do pass. Please elaborate!

Reply:
throws `InvalidOperationException` now if `null`